docs/workflow/tutorial-mobile-show-map-callout.mdx

```diff
@@ -246,7 +248,7 @@ namespace App1.Workflow
             // ArcGIS runtime Map
             Map map = mapProvider.Map;
             // ArcGIS runtime MapView
-            MapView mapView = mapProvider.GetType().GetProperty("MapView").GetValue(mapProvider, null) as MapView;
+            MapView mapView = (mapProvider as WorkflowMapProvider).View as MapView;
```

Comment:
You should always do a null check after doing a cast with the as operator. If you want to assert that the cast is expected to always be successful, you should use the traditional explicit cast expression (WorkflowMapProvider)mapProvider.

Reply:
Opted for the explicit cast.